tools/generate_report.py

```diff
@@ -77,16 +77,16 @@
 # Package Dictionaries
 pkgs = report["pkgs"]
 last_pkgs = last_report["pkgs"]
-pkgs_new = pkgs.keys() - last_pkgs.keys()
-pkgs_removed = last_pkgs.keys() - pkgs.keys()
+pkgs_new = sorted(pkgs.keys() - last_pkgs.keys())
+pkgs_removed = sorted(last_pkgs.keys() - pkgs.keys())
```

Comment:
Instead of sorting these here, perhaps `write_details_list` should sort the list of package names given to it?

Reply:
Yes, this would make more sense.